scenes/game/game.tscn

```diff
@@ -22,7 +22,7 @@
 [ext_resource type="PackedScene" uid="uid://dvan4a8ftuplj" path="res://scenes/card/character_card_base.tscn" id="18_imast"]
 [ext_resource type="PackedScene" uid="uid://w87ls4e8oasj" path="res://scenes/game/combat_log.tscn" id="18_k7mnw"]
 [ext_resource type="Texture2D" uid="uid://hj14gadm15wr" path="res://assets/icons/speechbubble.png" id="21_86687"]
-[ext_resource type="PackedScene" uid="uid://b8kmmrd5jmqdd" path="res://scenes/game/modal_dialog.tscn" id="21_qe5we"]
+[ext_resource type="PackedScene" path="res://scenes/game/modal_dialog.tscn" id="21_qe5we"]
```

Comment:
Can you try cleaning out your .imported or something?
Having this get deleted every time is annoying.

Reply:
I just tried removing the entire .godot folder, will see if it helps